release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3391,4 +3391,28 @@ $ oc adm release info 4.10.31 --pullspecs
 [id="ocp-4-10-31-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-32"]
+=== RHBA-2022:6372 - {product-title} 4.10.32 bug fix
+
+Issued: 2022-09-13
+
+{product-title} release 4.10.32 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:6372[RHBA-2022:6372] advisory. There are no RPM packages for this release.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.32 --pullspecs
+----
+
+[id="ocp-4-10-32-bug-fixes"]
+==== Bug fixes
+
+* Previously, dual-stack clusters with PROXY protocol were only enabled on IPv6 and not IPv4. With this update, {product-title} now enables PROXY protocol for both IPv6 and IPv4 on dual-stack clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2096362[*BZ#2096362*])
```

Comment:
```suggestion
* Previously, dual-stack clusters using the PROXY protocol only enabled it on IPv6 and not IPv4. With this update, {product-title} now enables the PROXY protocol for both IPv6 and IPv4 on dual-stack clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2096362[*BZ#2096362*])
```
Slight rewording. What do you think? Otherwise, LGTM.